index.css

```diff
@@ -0,0 +1,68 @@
+body{
+  margin: 0;
+}
+/*start page css*/
+.page{
+  width: 495px;
+  margin: auto;
+  text-align: center;
+}
+.page .header{
+  height: 50px;
+  border: 1px solid red;
+  padding: 2px 0 2px 20px;
+}
+.page .content{
+  border: 1px solid red;
+  min-height: 500px;
+}
+.page .footer{
+  height: 50px;
+  border: 1px solid red;
+}
+/*end page css*/
+
+/*start header css*/
+.header .header-left{
+  float: left;
+}
+.header .header-right{
+  float: right;
+}
+/*end header css*/
+
+/*start navbar css*/
+.navbar{
+  display: inline-flex;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+.navbar .item{
+  padding: 15px;
+  cursor: pointer;
+}
+.navbar .item.active{
+  border-bottom: 5px solid blue;
+}
+/*end navber css*/
```

Comment:
it will be better if we can put all empty css class here also.
someone can add more css rule if need in the future